test/integration.js

```diff
@@ -236,7 +242,7 @@ tape('POST /channel/{id}/{validator-messages}: wrong signature', function(t) {
 			"messages": [{ 
 				"type": 'NewState', 
 				stateRoot,
-				"balances": { "myAwesomePublisher" : "9" },
+				"balances": { "myAwesomePublisher" : "12", "anotherPublisher": "3" },
```

Comment:
why is this changed?

Reply:
The test case was throwing a different error than the required one. It was throwing `invalid transition error` instead of `invalid signature`